Internetmedicin.se

hkarn opened the pull request.

KrisGalcz commented:
It needs to be reviewed by @monzta because I'm not sure

```
internetmedicin.se###OutsiderArea
internetmedicin.se##[class^="adStartPage"]
internetmedicin.se##.LeftAdFrame 
||internetmedicin.se/imgCont/banner^$image
||internetmedicin.se/banner^$subdocument
```
these are blocking also IMjobs which are from the same company as the main page. As long as these are placed in the same way as ads and with class which contain "ad" I think these filters could stay

`icd.internetmedicin.se###tMytt`

This is blocking content on the left side

`icd.internetmedicin.se#?#div:-abp-contains(Annons)`

This one is too risky. This kind of filters working very generic. It would block whole div eg. in
`<div><p>Hey bleble. Annons which I saw last time was bleble</p></div>`
so it could block whole article with string "Annons"
you could change it for eg.
`icd.internetmedicin.se#?#td > div:-abp-contains(Annons)`

hkarn commented:
Fixed the abp-contains.

The also removed the ###tMytt filter. But to be sure is right here is some more info on what it contains.
It is just a copy of the list at
http://mednytt.internetmedicin.se
I have hidden some external news aggregates like this earlier usually labelled "sponsored articles" and such. There does seem to be slight weight towards medical articles in this one so maybe it does something more then just placing sponsored content. But 30-50% are just general news from the tabloid press, no relation to medicine.
